app/src/main/java/com/navapp/navigation/requests/RoutePlanner.java

```diff
@@ -0,0 +1,45 @@
+package com.navapp.navigation.requests;
```

Comment:
This is a `RoutePlanner`. Sounds like it can go into the `route` package then

Reply:
I guess it should be renamed

Reply:
yeah ok